Extension/ui/settings.html

```diff
@@ -0,0 +1,131 @@
+
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="Content-Security-Policy" content="default-src 'none'; img-src vscode-resource:; script-src 'nonce-{{nonce}}';">
+  </head>
+
+  <!-- todo: create .css for styles -->
+  <style>
+    th, td {
+      padding: 10px;
+      font-family: verdana, sans-serif;
+      font-size: 15px;
+    }
+  </style>
+
+  <body>
+
+  <!-- todo: change table into div elements -->
+  <table>
+
+  <!-- row 0 -->
+  <!-- Header -->
+  <tr>
+      <td>
+        <table>
+          <td>
+            <img src="{{root}}/ui/LanguageCCPP_color_128x.png" alt="Microsoft C/C++ Extension" title="Microsoft C/C++ Extension">
+          </td>
+          <td>
+               Microsoft <br>
+               C/C++ Extension
+          </td>
+        </table>
+      </td>
+  </tr>
+
+  <!-- row 1 -->
+  <!-- Save -->
+  <tr>
+      <td>
+          <p>Saves to active configuration <span id=activeConfig></span> in <a href="command:C_Cpp.ConfigurationEditJSON" title="Edit configurations in JSON file">c_cpp_properties.json</a></p>
+      </td>
+  </tr>
+
+  <!-- row 2 -->
+  <!-- compiler path -->
+    <tr>
+      <td>
+        Compiler path
+       <p>
+          <!--<span id="compilerPathInvalid">compiler path is invalid.</span><br>-->
+          <input type="text" id="compilerPath" size="54">

```

Comment:
Do we need to set a specific size for this?  A full path (plus args) may not fit in a small box.

Reply:
(though you can address this with the subsequent styling PR)

Reply:
Will address the size in styling PR